package.json

```diff
@@ -4,7 +4,7 @@
     "name": "Junior Rojas"
   },
   "scripts": {
-    "test": "export NODE_PATH=. && jest"
+    "test": "export NODE_PATH=. && jest ./test"
```

Comment:
If you run `npm test` in the root directory, it also runs the tests in `./demo/test`, which we want to ignore, we only want to run the tests in `./test`. Figure out how to fix this.